server/ServerApp.h

```diff
@@ -186,10 +186,14 @@ class ServerApp : public IApp {
       * are thereby victorious. */
     void    CheckForEmpireElimination();
 
-    /** Intializes single player game universe, sends out initial game state to
-      * clients, and signals clients to start first turn */
+    /** Intializes single player game universe.*/
     void    NewSPGameInit(const SinglePlayerSetupData& single_player_setup_data);
 
+    /** Verifies single player game AIs are compatible with already
+      * created universe. Sends out initial game state to
+      * clients, and signals clients to start first turn */
+    void    VerifySPGameAIs(const SinglePlayerSetupData& single_player_setup_data);
```

Comment:
I broke it into two functions, `bool VerifySPGameAIs()` and `SendNewGameStartMessages()` whose functions are more clearly implied by their names.